widget/playground/src/components/Slider/Slider.tsx

```diff
@@ -13,6 +13,7 @@ import {
 } from './Slider.styles';
 
 const MAX_VALUE = 100;
+const DEFAULT_COLOR = '#2284ED';
```

Comment:
considering that this value should be equal to the secondary color of 'DEFAULT_THEME_COLORS', please check if you can use it directly.

Reply:
fixed